src/module/api/invite/delete/index.ts

```diff
@@ -30,14 +33,12 @@ export async function Delete(IDeleteInvite: IDeleteInvite): Promise<IInvite[]> {
         reject(err);
         return;
       } else {
-        const invitesPb = res.getObjList();
-
-        const status = invitesPb.map((invitePb) => {
-          const metaPb = invitePb.getMetadataMap();
-          const id = metaPb.get(key.InviteStatus);
-          return id as unknown as IInvite; // TODO: check
```

Comment:
`key.*Status` has always been one of `updated`, `created`, or `deleted`. We ignored the return value so this wasn't noticed before. I changed this in every mutating API call.